lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_other_countries.govspeak.erb

```diff
@@ -10,7 +10,5 @@
                locals: {
                  overseas_passports_embassies: calculator.overseas_passports_embassies
                } %>
-  <% else %>
-    <% raise "The outcome for #{calculator.ceremony_country} is not handled" %>
```

Comment:
I think it does, @floehopper. I'll add a Trello card if/when I merge this branch.